2020-08-24 10:22:02 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:02 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:41" - wri_to_excel_cell_value - INFO - 写入数据成功,写入的测试数据是艾提科信科技有限公司
2020-08-24 10:22:02 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:42" - wri_to_excel_cell_value - INFO - 清除数据成功,清除的测试数据是

2020-08-24 10:22:10 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:10 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:41" - wri_to_excel_cell_value - INFO - 写入数据成功,写入的测试数据是华泰通安传媒有限公司
2020-08-24 10:22:10 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:42" - wri_to_excel_cell_value - INFO - 清除数据成功,清除的测试数据是

2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:36 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据

2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据

2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:38 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:44" - wri_to_excel_cell_value - ERROR - 写入数据失败，失败的原因是：[Errno 13] Permission denied: '../test_data/test_case.xls'
2020-08-24 10:22:39 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据

2020-08-24 10:23:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:23:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:41" - wri_to_excel_cell_value - INFO - 写入数据成功,写入的测试数据是数字100网络有限公司
2020-08-24 10:23:37 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:42" - wri_to_excel_cell_value - INFO - 清除数据成功,清除的测试数据是

2020-08-24 10:24:03 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:31" - wri_to_excel_cell_value - INFO - 开始写入测试数据
2020-08-24 10:24:03 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:41" - wri_to_excel_cell_value - INFO - 写入数据成功,写入的测试数据是新宇龙信息传媒有限公司
2020-08-24 10:24:03 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/samples/xlutils_faker_demo.py:42" - wri_to_excel_cell_value - INFO - 清除数据成功,清除的测试数据是

2020-08-24 11:34:48 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:64" - <module> - INFO - 

====================================================开始清除上次测试结果保留的值====================================================


2020-08-24 11:34:48 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:68" - <module> - ERROR - 

===================================================清除上次测试结果保留的值失败，失败的原因是：[Errno 13] Permission denied: 'C:\\Users\\Administrator\\PycharmProjects\\newadmin_api_framework\\common\\../test_data/test_case.xls'==========================================


2020-08-24 11:34:48 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:51" - run - INFO - 

====================================================创建测试报告文件路径成功====================================================


2020-08-24 11:34:48 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\api_testcase\api_test.py:16" - <module> - INFO - 

====================================================正在获取测试用例数据====================================================


2020-08-24 11:34:48 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:40" - load_test_suite - INFO - 

====================================================加载所有测试用例完毕====================================================


2020-08-24 11:34:48 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:41" - load_test_suite - INFO - 

====================================================开始执行所有测试用例====================================================


2020-08-24 11:34:49 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:77" - __post - INFO - 登录后台接口接口调用成功
2020-08-24 11:34:49 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:78" - __post - INFO - 
正在执行的测试用例名称：测试能否正确登录后台接口
正在请求的接口名称：登录后台接口
测试用例编号：case01
正在执行的测试用例步骤：step_01
请求方式：post
请求地址是：/admin/login
请求参数：{"loginName":"liseen1","password":"000000"}
请求头：{"Accept":"application/json, text/plain, */*","Content-Type": "application/json;charset=UTF-8","User-Agent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36","Accept-Language": "zh-CN,zh;q=0.9,en;q=0.8","Accept-Encoding":"gzip, deflate","Connection": "keep-alive"}


2020-08-24 11:34:53 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:77" - __post - INFO - 登录后台接口接口调用成功
2020-08-24 11:34:53 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:78" - __post - INFO - 
正在执行的测试用例名称：测试能否正确查询平台商管理接口
正在请求的接口名称：登录后台接口
测试用例编号：case02
正在执行的测试用例步骤：step_01
请求方式：post
请求地址是：/admin/login
请求参数：{"loginName":"liseen1","password":"000000"}
请求头：{"Accept":"application/json, text/plain, */*","Content-Type": "application/json;charset=UTF-8","User-Agent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36","Accept-Language": "zh-CN,zh;q=0.9,en;q=0.8","Accept-Encoding":"gzip, deflate","Connection": "keep-alive"}


2020-08-24 11:34:55 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:72" - <module> - INFO - 

===================================================已执行完所有测试用例，测试活动结束==========================================


2020-08-24 11:34:55 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:77" - <module> - INFO - 

===================================================测试活动已结束，是否发送报告===================================================



2020-08-24 11:35:07 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:85" - <module> - INFO - 各位大佬不需要测试报告结果，没有发送测试报告

2020-08-24 11:35:40 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:64" - <module> - INFO - 

====================================================开始清除上次测试结果保留的值====================================================


2020-08-24 11:35:40 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:66" - <module> - INFO - 

====================================================开始清除上次测试结果保留的值成功====================================================


2020-08-24 11:35:40 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:51" - run - INFO - 

====================================================创建测试报告文件路径成功====================================================


2020-08-24 11:35:40 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\api_testcase\api_test.py:16" - <module> - INFO - 

====================================================正在获取测试用例数据====================================================


2020-08-24 11:35:40 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:40" - load_test_suite - INFO - 

====================================================加载所有测试用例完毕====================================================


2020-08-24 11:35:40 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:41" - load_test_suite - INFO - 

====================================================开始执行所有测试用例====================================================


2020-08-24 11:35:41 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:77" - __post - INFO - 登录后台接口接口调用成功
2020-08-24 11:35:41 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:78" - __post - INFO - 
正在执行的测试用例名称：测试能否正确登录后台接口
正在请求的接口名称：登录后台接口
测试用例编号：case01
正在执行的测试用例步骤：step_01
请求方式：post
请求地址是：/admin/login
请求参数：{"loginName":"liseen1","password":"000000"}
请求头：{"Accept":"application/json, text/plain, */*","Content-Type": "application/json;charset=UTF-8","User-Agent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36","Accept-Language": "zh-CN,zh;q=0.9,en;q=0.8","Accept-Encoding":"gzip, deflate","Connection": "keep-alive"}


2020-08-24 11:35:44 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:77" - __post - INFO - 登录后台接口接口调用成功
2020-08-24 11:35:44 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:78" - __post - INFO - 
正在执行的测试用例名称：测试能否正确查询平台商管理接口
正在请求的接口名称：登录后台接口
测试用例编号：case02
正在执行的测试用例步骤：step_01
请求方式：post
请求地址是：/admin/login
请求参数：{"loginName":"liseen1","password":"000000"}
请求头：{"Accept":"application/json, text/plain, */*","Content-Type": "application/json;charset=UTF-8","User-Agent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36","Accept-Language": "zh-CN,zh;q=0.9,en;q=0.8","Accept-Encoding":"gzip, deflate","Connection": "keep-alive"}


2020-08-24 11:35:46 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:77" - __post - INFO - 查询平台商管理接口接口调用成功
2020-08-24 11:35:46 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:78" - __post - INFO - 
正在执行的测试用例名称：测试能否正确查询平台商管理接口
正在请求的接口名称：查询平台商管理接口
测试用例编号：case02
正在执行的测试用例步骤：step_02
请求方式：post
请求地址是：/admin/platform/cagent/listPage
请求参数：{"page":1,"limit":10,"cagent":"","status":""}
请求头：{"Accept":"application/json, text/plain, */*","Content-Type": "application/json;charset=UTF-8","User-Agent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36","Accept-Language": "zh-CN,zh;q=0.9,en;q=0.8","Accept-Encoding":"gzip, deflate","Connection": "keep-alive","X-Token": "XhJpQQSFUeLL6KADQP8ICVqvqXWZFuldPHOBDuhJY6xLWaeYr+J1LGmyfbJwD3txhSSTRhqKSLe11BL6Xca04Lbp8i+olYp4cGohHA9cOyYunlmbp4GvvNZqp8yVetn0gIwzabq6f5t1sRP9TSsEtGaNoAnAjh7e8QxBGqK+rFY/lJGQZQj63xWA5l6p+Xlf1TTRJR6pQtAVu4p0YRrf93xbiRkAZWbJE18e/sOaBsL7UrTieDMu1W8QXNoMA5HaAaQMNLdSG2n1yZ3DmO8vORhf/6uu3ogjPKgOtM4m2yQIeaRQ3o0aNWG75d40cCW2XGFr5tW9G9/FdYdXGR2NHQMt+34YCDBRd4of4bKJ7wvJRgNeZlSGEad6usB0GaJ5+pL9jIKgW6P9yeJ47k035pUpTUjs5Qyx64Hi8lf8qJ+huOJO2gUiX0VOvLmUkbf1njzyQ6lMx2SkgW1TPI+733HwQImoTYfEA2fWRSU909A+lBmAxRIXQrNu81XIKxNV","sid":"0","terminal":"0"}


2020-08-24 11:35:51 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:77" - __post - INFO - 登录后台接口接口调用成功
2020-08-24 11:35:51 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:78" - __post - INFO - 
正在执行的测试用例名称：验证能正常查询金积分商城类别
正在请求的接口名称：登录后台接口
测试用例编号：case02
正在执行的测试用例步骤：step_01
请求方式：post
请求地址是：/admin/login
请求参数：{"loginName":"liseen1","password":"000000"}
请求头：{"Accept":"application/json, text/plain, */*","Content-Type": "application/json;charset=UTF-8","User-Agent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36","Accept-Language": "zh-CN,zh;q=0.9,en;q=0.8","Accept-Encoding":"gzip, deflate","Connection": "keep-alive"}


2020-08-24 11:35:54 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:77" - __post - INFO - 查询积分商城类别接口调用成功
2020-08-24 11:35:54 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:78" - __post - INFO - 
正在执行的测试用例名称：验证能正常查询金积分商城类别
正在请求的接口名称：查询积分商城类别
测试用例编号：case02
正在执行的测试用例步骤：step_01
请求方式：post
请求地址是：
请求参数：{"page":1,"limit":10,"cid":""}
请求头：{"Accept":"application/json, text/plain, */*","Content-Type": "application/json;charset=UTF-8","User-Agent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36","Accept-Language": "zh-CN,zh;q=0.9,en;q=0.8","Accept-Encoding":"gzip, deflate","Connection": "keep-alive","X-Token": "NKsBl1f3X1SXP9uwO1F9irLWTW7j/NCcbTNza4yMdnWNeqoaJaZecKWxe+q7Ln6WDpiSUKcXmVZxFqhcUnBmR5wPlnppkRrgmq1M2RbGwYhsZDniXGwKbOunoktZ5SzcLrwnt0BT8VLZyUU/3w6p4OrpSQP3SkV9hGpTjVWLxJuI/eGLcNu8Qr5SnzkL4VjhWCFF/DwK9s7iWGhwVgovC3lR+jcW/at5IKGdBFoo0nd3s2C2xKKDxsZ5kqNjr8TUHKum8CBpqMTvmU1+UJC/PL3ptSFTio4Bg57++Fsyxu2Us/pl2jNU8AfFnHV1BAswtEN66WPoz4lGBdw48yFKn1I2zhCRqgpYxgtKYSidM0H86xAkt2SOPwQzfy6oZyY8D3MzaCnqY+Zo3JxIAQwo/m0i02ie1awvPZzWUBFwrFU38reotRukfvXefQ3m8ftxvNaO86yxnGgL8MjZpgA0fjhNcCmBVERBPbRH7A+3KuL62oc2PlLcNPxlxr5DiGaj","sid":"0","terminal":"0"}


2020-08-24 11:35:59 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:72" - <module> - INFO - 

===================================================已执行完所有测试用例，测试活动结束==========================================


2020-08-24 11:35:59 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:77" - <module> - INFO - 

===================================================测试活动已结束，是否发送报告===================================================



2020-08-24 11:36:05 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:85" - <module> - INFO - 各位大佬不需要测试报告结果，没有发送测试报告

2020-08-24 11:36:59 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:64" - <module> - INFO - 

====================================================开始清除上次测试结果保留的值====================================================


2020-08-24 11:36:59 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:66" - <module> - INFO - 

====================================================开始清除上次测试结果保留的值成功====================================================


2020-08-24 11:36:59 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:51" - run - INFO - 

====================================================创建测试报告文件路径成功====================================================


2020-08-24 11:36:59 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\api_testcase\api_test.py:16" - <module> - INFO - 

====================================================正在获取测试用例数据====================================================


2020-08-24 11:36:59 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:40" - load_test_suite - INFO - 

====================================================加载所有测试用例完毕====================================================


2020-08-24 11:36:59 - 接口自动化日志 - "C:/Users/Administrator/PycharmProjects/newadmin_api_framework/test_runner/run_case.py:41" - load_test_suite - INFO - 

====================================================开始执行所有测试用例====================================================


2020-08-24 11:37:00 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:77" - __post - INFO - 登录后台接口接口调用成功
2020-08-24 11:37:00 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:78" - __post - INFO - 
正在执行的测试用例名称：测试能否正确登录后台接口
正在请求的接口名称：登录后台接口
测试用例编号：case01
正在执行的测试用例步骤：step_01
请求方式：post
请求地址是：/admin/login
请求参数：{"loginName":"liseen1","password":"000000"}
请求头：{"Accept":"application/json, text/plain, */*","Content-Type": "application/json;charset=UTF-8","User-Agent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36","Accept-Language": "zh-CN,zh;q=0.9,en;q=0.8","Accept-Encoding":"gzip, deflate","Connection": "keep-alive"}


2020-08-24 11:37:02 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:77" - __post - INFO - 登录后台接口接口调用成功
2020-08-24 11:37:02 - 接口自动化日志 - "C:\Users\Administrator\PycharmProjects\newadmin_api_framework\common\requests_utils.py:78" - __post - INFO - 
正在执行的测试用例名称：测试能否正确查询平台商管理接口
正在请求的接口名称：登录后台接口
测试用例编号：case02
正在执行的测试用例步骤：step_01
请求方式：post
请求地址是：/admin/login
请求参数：{"loginName":"liseen1","password":"000000"}
请求头：{"Accept":"application/json, text/plain, */*","Content-Type": "application/json;charset=UTF-8","User-Agent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36","Accept-Language": "zh-CN,zh;q=0.9,en;q=0.8","Accept-Encoding":"gzip, deflate","Connection": "keep-alive"}


